### PR TITLE
feature/7876-theo-HomeScreenButtonComponent

### PR DESCRIPTION
**Pull request by theodur**
## Description of Change

Creates a new component for the home screen modules.

**Note**: The Activity section itself will be handled in a different ticket.

## Screenshots/Video

Android/iOS: 

&nbsp;&nbsp;

&nbsp;&nbsp;

## Testing

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations

Verify that the component matches the [Figma designs](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/7876).

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/ios/VAMobile/Info.plist [hunk 1]
```diff
@@ -66,6 +66,7 @@
 		<string>SourceSansPro-Italic.ttf</string>
 		<string>SourceSansPro-Regular.ttf</string>
 		<string>SourceSansPro-SemiBold.ttf</string>
+		<string>Bitter-Regular.ttf</string>
```
Comment (dumathane) [inline VAMobile/ios/VAMobile/Info.plist:69]: What made you do this?

**Review by dumathane — APPROVED** [review]
Looks good. Still need to pull it on the emulator and muck around.

**Comment by TKDickson** [comment]
Found some issues (documented in ticket), sending back for changes.

**Comment by TKDickson** [comment]
A few VQA changes from Lauren in the ticket

**Comment by dumathane** [comment]
This is waiting on Laurens sign off and Theo to confirm the font styling is working as intended.

**Review by TKDickson — APPROVED** [review]
QA and VQA signoff complete, good to merge into XL

(we'll just put the "pull the number of days for the appointments button from the backend" onto some other PR, I'm TOO EXCITED not to merge this one right now)

**Pull request merged**